fiftyone/core/fields.py

```diff
@@ -86,11 +96,26 @@ def validate_constraints(ftype=None, embedded_doc_type=None, read_only=None):
         if not isinstance(read_only, bool):
             raise ValueError("read_only must be a boolean")
 
+    if info_keys is not None:
+        has_contraints = True
+        if not isinstance(info_keys, (list, str)):
+            raise ValueError("info_keys must be a single or a list of str's")
+
+    if created_after is not None:
+        has_contraints = True
+        if not isinstance(created_after, datetime):
+            raise ValueError("created_after must be a datetime")
+
     return has_contraints
```

Comment:
How about: `return (ftype or embedded_doc_type or readonly or info_keys or created_after or created_after) is not None`?  We can get rid of these `has_constraints = True` statements